pkg/controllermanager/controller/shoot/shoot_control_reconcile.go

```diff
@@ -278,15 +278,25 @@ func (c *Controller) runReconcileShootFlow(o *operation.Operation, operationType
 			Fn:           flow.TaskFn(botanist.HibernateControlPlane).RetryUntilTimeout(defaultInterval, 2*time.Minute).DoIf(o.Shoot.HibernationEnabled),
 			Dependencies: flow.NewTaskIDs(initializeShootClients, deploySeedMonitoring, deploySeedLogging, deployClusterAutoscaler),
 		})
-		deployExtensionResource = g.Add(flow.Task{
+		deployExtensionResources = g.Add(flow.Task{
 			Name:         "Deploying extension resources",
-			Fn:           flow.TaskFn(botanist.DeployExtensionResources).RetryUntilTimeout(defaultInterval, defaultTimeout),
+			Fn:           flow.TaskFn(botanist.DeployExtensionResources).RetryUntilTimeout(defaultInterval, defaultTimeout).SkipIf(o.Shoot.HibernationEnabled),
```

Comment:
Why is that only done if the shoot is not hibernated? I think we have to do it always in order to give extension controllers the chance to do whatever is necessary if the shoot gets hibernated?